styles.css

```diff
@@ -77,3 +88,160 @@ h2 {
   justify-content: space-between;
   margin-left: 30px;
 }
+
+.work-container {
+  display: grid;
+  width: 100vw;
+  row-gap: 15vh;
+  place-items: center;
+  background-color: #f6f6f9;
+}
+
+.work-item {
+  place-self: center;
+  width: 90vw;
+  border: 2px solid rgb(197, 195, 195);
+  border-radius: 20px;
+  background-color: white;
+}
```

Comment:
- [ ] I believe this is the class used for every project. I think it will be good to use display flex to align the content of each project